### PR TITLE
Upgradable L1TokenGateway and L2TokenGateway

### PR DESCRIPTION
**Pull request by telome**

### CODE REVIEW

File: src/L1TokenGateway.sol [hunk 1]
```diff
@@ -27,19 +28,20 @@ interface TokenLike {
...
+    uint256 public isOpen;
     address public escrow;
 
-    // --- immutables ---
+    // --- immutables and const ---
 
     address public immutable counterpartGateway;
     address public immutable l1Router;
     address public immutable inbox;
+    string public constant version = "1";
```
Comment (sunbreak1211) [inline src/L1TokenGateway.sol]: I'd say to align it in this case.

File: src/L2TokenGateway.sol [hunk 2]
```diff
@@ -27,18 +28,19 @@ interface TokenLike {
...
     mapping(address => uint256) public maxWithdraws;
-    uint256 public isOpen = 1;
+    uint256 public isOpen;
 
-    // --- immutables ---
+    // --- immutables and const---
 
     address public immutable l2Router;
     address public immutable counterpartGateway;
+    string public constant version = "1";
```
Comment (sunbreak1211) [inline src/L2TokenGateway.sol]: Same here

**Review by oldchili — APPROVED** [review]
LGTM

**Review by sunbreak1211 — APPROVED** [review]
Looks good

**Pull request merged**